### PR TITLE
Feature/issue 34/bird view

### LINKED ISSUE #34: Bird view for the overview of the graph
The bird view should allow user to see the overview of the whole graph as well as to see where exactly he is positioned in the whole graph. Represented by a frame, the birdview will have a direct connection and observation of the user's movement through the graph

### PR DESCRIPTION
**Pull request by trami25**
closes #34  The complete birdView which contains an overview of all the nodes of the graph and its edges. It supports the tracker which shows the user where he is located inside of the graph. It actively follows what is happening in the main view and it behaves accordingly.

### CODE REVIEW

**Review by ZivanicBojan — APPROVED** [review]
Great job!

File: graph_visualizer_platform/src/graph_visualizer_platform/BirdView.py
Comment (vradovic) [inline graph_visualizer_platform/src/graph_visualizer_platform/BirdView.py:1]: We don't need this.

**Review by vradovic — CHANGES_REQUESTED** [review]
Looks good, but the data in the provider should not be commented.

**Review by vradovic — APPROVED** [review]
LGTM

**Pull request merged**